vaadin-date-picker-flow/src/test/java/com/vaadin/flow/component/datepicker/DatePickerTest.java

```diff
@@ -111,4 +111,19 @@ public void openAndClose_openedPropertyIsUpdated() {
         assertFalse(OPENED_PROPERTY_NOT_UPDATED, picker.isOpened());
     }
 
+    @Test
+    public void clearButtonVisiblePropertyValue() {
+        DatePicker picker = new DatePicker();
+
```

Comment:
Sorry, I forgot to say this earlier, but checking the default value would be nice to test just in case.

Reply:
Done.